setup.py

```diff
@@ -61,7 +61,8 @@ def pip_requirements(fname='requirements.txt'):
             'cf-1.7 = compliance_checker.cf.cf:CF1_7Check',
             'acdd-1.1 = compliance_checker.acdd:ACDD1_1Check',
             'acdd-1.3 = compliance_checker.acdd:ACDD1_3Check',
-            'ioos_sos = compliance_checker.ioos:IOOSBaseSOSCheck',
+            'ioos_sosGC = compliance_checker.ioos:IOOSSOSGCCheck',
```

Comment:
We may want to keep a special case for the IOOSBaseSOSCheck since the original behavior was to generate the GetCapabilities or DescribeSensor when pointed at an arbitrary endpoint.